cmd/go-tpc/misc.go

```diff
@@ -49,6 +49,8 @@ func execute(ctx context.Context, w workload.Workloader, action string, index in
 		return w.Cleanup(ctx, index)
 	case "check":
 		return w.Check(ctx, index)
+	case "schema":
```

Comment:
maybe not add an option for schema, create schema by default?

Reply:
Not sure. So that when you run `go-tpc` to generate csv files, you still need to provide db information? If I don't have a database currently and I only want to generate csv files, how do you handle this case?

Reply:
I suggest that you could use `w.db == nil` to decide whether should createTables here, but you should change a little code to openDB:

https://github.com/pingcap/go-tpc/blob/4be1fd1a0d14e7a6ae8e60d8c2385eb61ec74f5b/cmd/go-tpc/tpcc.go#L23-L31

Eg, if a user keys such command, go-tpc could create schema automatically for user.

```
./bin/go-tpc tpcc --warehouses 4 prepare --host 127.0.0.1 --csv.output data --csv.table history --csv.table orders
```

Reply:
Provides the `schema` subcommand is also ok for me, but we can make it easy on some cases.